includes/class-coblocks-form.php

```diff
@@ -973,7 +974,7 @@ public function process_form_submission( $atts ) {
 		$email_headers = (array) apply_filters(
 			'coblocks_form_email_headers',
 			array(
-				"Reply-To: {$reply_to}",
+				"Reply-To: {$sender_name} <{$sender_email}>",
```

Comment:
@EvanHerman What happens if one or more of these vars is empty? Just curious what the outcome is.

Reply:
@EvanHerman For instance, a name without an email is useless for a reply to field.

Reply:
@fjarrett-godaddy Good question. I thought there was a catch in there that if the email field isn't present the form wouldn't submit. There isn't one, so it looks like in that instance the `sender_email` would be empty, and if the name wasn't present that would also be empty and the `Reply-To` field would just be empty.

Maybe we can sidebar this in standup to come up with a reasonable solution. Maybe if those aren't set then we just don't pass a `Reply-To` header and let `wp_mail()` handle it.

Reply:
@EvanHerman Yeah I think as long as there is a `$sender_email` then we should include the Reply-To header, otherwise we send the message with default headers. The `$sender_name` is a bonus, if we have it we should include it.